### PR TITLE
code styles

### PR DESCRIPTION
**Pull request by cliffordp**
[ET-843] but not really

[ET-843]: https://moderntribe.atlassian.net/browse/ET-843

### CODE REVIEW

File: src/views/registration-js/attendees/fields/birth.php [hunk 1]
```diff
@@ -6,13 +6,17 @@
...
+ * @version TBD
+ *
+ * @since 4.12.1 Introduced template.
+ * @since TBD Fix PHP notice for undefined variable.
  *
  * @see     Tribe__Tickets_Plus__Meta__Field__Birth
  */
 
-$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$required    = isset( $field->required ) && 'on' === $field->required;
```
Comment (tr1b0t) [inline src/views/registration-js/attendees/fields/birth.php:17]: :warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 4 spaces

<sub>hash: c4cfe3f588068a6ad31c641ea85f94d3</sub>

File: src/views/shortcodes/my-attendance-list.php [hunk 2]
```diff
@@ -6,22 +6,42 @@
...
-				<?php echo get_the_title( $id ); ?>
+	<?php
+	foreach ( $event_ids as $event_id ) :
+		$is_event               = function_exists( 'tribe_is_event' ) ? tribe_is_event( $event_id ) : false;
+		$direct_link_my_tickets = $view->get_tickets_page_url( $event_id, $is_event );
+		?>
+		<?php $start_date = tribe_get_start_date( $event_id ); ?>
+		<li class="event-<?php echo esc_attr( $event_id ); ?>">
+			<a href="<?php echo esc_url( get_permalink( $event_id ) ); ?>" class="event-post-link">
+				<?php echo get_the_title( $event_id ); ?>
```
Comment (tr1b0t) [inline src/views/shortcodes/my-attendance-list.php:30]: :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_the_title'.

<sub>hash: 83dfefc66d1105b831e5f3605ea4d0cc</sub>

**Comment by cliffordp** [comment]
@sc0ttkclark please merge this as none of my code would have affected failed build (as a result of syncing up with base branch)

File: src/views/tickets/orders.php [hunk 3]
```diff
@@ -31,7 +31,7 @@
 $post_type                  = get_post_type_object( $event->post_type );
 $user_id                    = get_current_user_id();
 $provider_id                = Tribe__Tickets__Tickets::get_event_ticket_provider( $event_id );
-$provider                   = call_user_func( array( $provider_id, 'get_instance' ) );
+$provider                   = call_user_func( [ $provider_id, 'get_instance' ] );
```
Comment (sc0ttkclark) [inline src/views/tickets/orders.php:34]: Any change in a view requires a `@since` / `@version` update.

Comment (cliffordp) [reply src/views/tickets/orders.php:34]: OK, i'll back this out then. I'd prefer less noise in the template's changelog for just one array() to []

**Pull request closed without merging**